Gemfile.lock

```diff
@@ -326,7 +324,12 @@ GEM
     notiffany (0.1.3)
       nenv (~> 0.1)
       shellany (~> 0.0)
-    oauth (0.5.8)
+    oauth (1.1.0)
```

Comment:
where did this upgrade come from?

Reply:
Probably because I `bundle update`d (Rails got updated to what Dependabot suggested as well)